File: snapshots/input/syntax/Main/Packages.cs
using System.Diagnostics.CodeAnalysis;
using DiffPlex.DiffBuilder;
using DiffPlex.DiffBuilder.Model;

namespace Main;

[SuppressMessage("ReSharper", "all")]
public class Packages
{
    DiffPaneModel Diff()
    {
        return InlineDiffBuilder.Diff("a", "b");
    }
}

File: snapshots/input/syntax/VBMain/Packages.vb
Imports System.Diagnostics.CodeAnalysis
Imports DiffPlex.DiffBuilder
Imports DiffPlex.DiffBuilder.Model

Namespace VBMain
    <SuppressMessage("ReSharper", "all")>
    Public Class Packages
        Private Function Diff() As DiffPaneModel
            Return InlineDiffBuilder.Diff("a", "b")
        End Function
    End Class
End Namespace
